text/00110-compatibility-strategy.md

```diff
@@ -0,0 +1,364 @@
+---
+feature name: compatibility-strategy
+start date: February 2nd, 2020
+rfc pr: https://github.com/aws/aws-cdk-rfcs/pull/111
+related issue: https://github.com/aws/aws-cdk-rfcs/issues/110
+---
+
+<!-- replace the blockquoted sections with your content -->
+# Summary
+
+There are currently two statements we advertise to our customers with regards to framework and cli compatibility:
```

Comment:
I don't know that we officially advertise this (but we should). Maybe make sure we have that in the user guide and if not, add an issue.

Reply:
Taking a note to check and create the issue if needed